### DISCUSSION TITLE
Is there a way to get ahold of you?

### DISCUSSION THREAD

**Discussion by slothtechtv** [madMAx43v3r/chia-plotter#120] [General]
Hey @madMAx43v3r -- is there a way to get ahold of you via email? I can't seem to find any contact info for you.

alex@slothtechtv.com

-Alex

**Comment by madMAx43v3r** [comment]
check commit logs `git log`

**Comment by Hardcore-fs** [comment]
don't do it... keep a low profile......

**Comment by madMAx43v3r** [comment]
lol he already made a video on youtube

**Comment by Hardcore-fs** [comment]
you do that..
Then you want an "appearance fee" or "consulting fee"
Even call girls charge for their services....
